apps/tup-cms/project.json

```diff
@@ -0,0 +1,49 @@
+{
+  "root": "apps/tup-cms/",
+  "sourceRoot": "apps/tup-cms/src/",
+  "projectType": "application",
+  "targets": {
+    "build": {
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": [
+          "npx nx build tup-ui",
```

Comment:
Should we also add to the build toolchain for tup-ui's project.json that it should build core styles?

Reply:
I think Nx already does this because it automatically crawls javascript dependencies, we need to do it more explicitly with tup-cms because it's a non-js app.

Reply:
It does not. If I delete `libs/core-styles/dist` and `dist/libs/core-styles`, then run `nx build tup-ui` (I have `nx` installed globally), I get an error about unavailability of a core-styles file. Running `nx build core-styles`, then `nx build tup-ui` fixes the error.

<details>

```
> nx run tup-ui:build

vite v2.9.14 building for production...
transforming...
✓ 107 modules transformed.
[vite:css] ENOENT: no such file or directory, stat '/Users/wbomar/Developer/Code/tup-ui/libs/core-styles/dist/settings/border.css'
file: /Users/wbomar/Developer/Code/tup-ui/apps/tup-ui/src/components/auth/LoginComponent/LoginComponent.module.css?used
error during build:
Error: ENOENT: no such file or directory, stat '/Users/wbomar/Developer/Code/tup-ui/libs/core-styles/dist/settings/border.css'
Warning: run-commands command "npx vite build --emptyOutDir" exited with non-zero status code
```

</details>

Reply:
I've created a ticket—[FP-1779](https://jira.tacc.utexas.edu/browse/FP-1779)—in case we need to punt this. But I expect it to eventually be a problem when automating deploy.